modules/hcp-update-using-mce-console.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * hosted_control_planes/hcp-updating.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="hcp-update-using-mce-console_{context}"]
+= Updating a hosted cluster by using the {mce-short} console
+
+You can update your hosted cluster by using the {mce-short} console.
+
+.Procedure
+
+. Select *All clusters*.
```

Comment:
Will it be obvious to the MCE user where they are starting (where they find this option)?

Reply:
Yes. It's the first option they will click on. Very easy to find.